rss-client/src/app/Account/components/account-settings-page/account-settings-page.component.ts

```diff
@@ -131,15 +135,31 @@ export class AccountSettingsPageComponent implements OnInit {
   }
 
   createAccount(event) {
+    let index = null;
     if (event == 'Eval') {
       let num = this.accounts[1].accTypeId;
       this.myAccount.accTypeId = num;
       this.myAccount.userId = this.userservice.userPersistance().userId;
+      index = 1;
     } else if (event == 'Bug') {
       this.myAccount.accTypeId = this.accounts[0].accTypeId;
       this.myAccount.userId = this.userservice.userPersistance().userId;
+      index = 0;
+    }
+    else if (event == 'Overflow'){
+      if(this.accounts[2] == undefined || this.accounts[2] == null){
+        alert("There is not a Rev Overflow account linked with your user");
```

Comment:
Where should a user go to link a Rev Overflow account to their user account on RSS?